### ISSUE TITLE
Series expansion of `airyai` at infinity undefined

### ISSUE THREAD

**Issue by oscarbenjamin** [sympy/sympy#23563]
A colleague asked if SymPy could handle this but apparently it cannot:
```python
In [13]: series(airyai(x), x, oo)
---------------------------------------------------------------------------
PoleError: 
Asymptotic expansion of airyai around [oo] is not implemented.
> /home/oscar/current/sympy/sympy.git/sympy/core/function.py(658)_eval_aseries()
    656         """
    657         from sympy.utilities.misc import filldedent
--> 658         raise PoleError(filldedent('''
    659             Asymptotic expansion of %s around %s is
    660             not implemented.''' % (type(self), args0)))
```
Also I don't know what this is:
```python
In [14]: series(airyai(x), x, zoo)
Out[14]: Ai(zoo)

In [15]: series(exp(x), x, zoo)
Out[15]: nan
```
Should expansion at `zoo` be the same as at `oo`?

**Comment by jksuom** [comment]
> Should expansion at zoo be the same as at oo?

Not necessarily, I think. Most special functions have an essential singularity at `zoo`. Then there is no series expansion at `zoo`, and in that case SymPy will currently (often) return something symbolic and unevaluated. That may not be the best result though. I think that the proper response would be to raise PoleError (or maybe rather some new kind of error related to essential singularity). On the other hand, there might be an asymptotic expansion at `oo`, in which case `series` could return (some terms of) that.
```
In [22]: series(1/x, x, zoo)
Out[22]: 0
```
This seems wrong to me. `1/x` has no singularity at `zoo`, hence the Laurent series `1/x` should be returned.

**Comment by anutosh491** [comment]
Well not only at `1/x` I think we might get wrong results in many of the `series(f(x), x, zoo)` cases.

Current series methods doesn't deal with `zoo` as it does it `oo/-oo`.
For -oo/oo , we are trying the inversion step which would work most places
https://github.com/sympy/sympy/blob/b1cf21e12e84ecef87b45091e3a524b657033ad9/sympy/core/expr.py#L2956-L2962

But for `zoo`, it follows the general path, no special treatment

Hence lets consider `(1/x).series(x, zoo)` ...... . This changes to `(1/x + zoo).series(x, 0)`, then during an intermediate step (line 2988) `x` is replaced with a positive dummy variable we declared , so we encounter something like this
```
>>> x = Symbol('x', positive=True)
>>> (1/(x + zoo))
0
```
@jksuom I guess that not how `zoo` should be handled , I could think of some approach here but would like to hear if have something to say .
Also speaking about the `airyai` function , I have a couple of bug fixes during the later half of my Gsoc proposed work which are quite closely related to asymptotic expansion of airyai function . I'll try to approach the error reported here at that time if no one catches up on it till then !
